specification/basic/basic.tex

```diff
@@ -136,29 +131,29 @@ \subsubsection{Regulation}\label{sec:basic_concepts_data_type_regulation}
     &
     Definitions that are not available to anyone except their authors.\newline
     Fixed port identifiers are permitted (although not recommended);
-    they are called \emph{``unregulated fixed port-ID''}.
+    they are called \emph{unregulated fixed port-ID}.
     \\
 \end{UAVCANSimpleTable}
 
 DSDL processing tools shall prohibit unregulated fixed port identifiers by default,
 unless they are explicitly configured otherwise.
 
-Each of the two sets of valid port identifiers (which are subject identifiers and service identifiers) are
-segregated into three categories (the ranges are documented in chapter \ref{sec:application_layer}):
+Each of the two sets of port identifiers (which are subject identifiers and service identifiers) are
+segregated into three categories:
 
 \begin{itemize}
     \item Application-specific port identifiers.
-    These can be assigned by changing relevant configuration parameters of the connected nodes
-    (in which case they are called \emph{non-fixed} or runtime-assigned),
-    or at the data type definition time (in which case they are called \emph{fixed unregulated},
-    and they generally should be avoided due to the risks of collisions as explained earlier).
+          These can be assigned by changing relevant configuration parameters of the connected nodes
+          (in which case they are called \emph{non-fixed} or runtime-assigned),
```

Comment:
I don't think so. The fact that an identifier is not glued to a data type does not mean that it is entirely volatile, so the possibility of a static bus load analysis is not necessarily lost. It could be lost if the identifiers were assigned arbitrarily without following any specific policy, but this seems evident and is probably outside of the scope of the specification. I think we should remove the chance of misunderstanding here by just dropping the phrase "runtime-assigned".

It is not the intention here to limit the functionality or the level of guarantees available for non-fixed IDs since they are essential for the protocol (and are among the most important changes compared to v0).